Symfony/CS/Tests/Fixer/PSR2/BracesFixerTest.php

```diff
@@ -1351,6 +1355,10 @@ function a()
         echo $foo;
     };
 
+    $b = function (int $foo) use($bar): string {
```

Comment:
please move the `{` to new line for both lambdas to check if they are moved to correct place